xcode/Subconscious/Shared/Components/Common/MarkupTextViewRepresentable.swift

```diff
@@ -123,16 +123,34 @@ struct MarkupTextViewRepresentable: UIViewRepresentable {
                 )
             )
         }
+        
+        // This allows us to intercept touches on embedded transclude blocks
+        override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+            guard let touch = touches.first else { return }
+            let tapPoint = touch.location(in: self)
+            
+            let textContentStorage = self.textLayoutManager?.textContentManager as! NSTextContentStorage
```

Comment:
Might be worth passing a reference to `NSTextContentStorage` explicitly, instead of this assert-y, cast-y approach.